### ISSUE TITLE
Error adding authenticator

### ISSUE THREAD

**Issue by Zeprix** [Jessecar96/SteamDesktopAuthenticator#1060]
I use a new gmail and when setting up it asks for a code. It logs in successfully but when it needs to add the authenticator it gives me this error.

**Comment by dyc3** [comment]
This is a known issue, and SDA isn't going to get updated for a while. Jessecar96 themselves has said on discord that he's too busy to update SDA right now. I highly recommend that you should use the steam mobile app, if you are able to.

If you are interested, I maintain steamguard-cli. It's capable of reading the exactly the same maFiles that SDA does, so you don't have to deal with a trade ban. It doesn't have a UI, so if you aren't really comfortable using a terminal, I would recommend you use the steam mobile app. You can download it here: https://github.com/dyc3/steamguard-cli/releases

Read the quickstart here: https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md

You can transfer an existing authenticator to the official steam mobile app, but that does cause a 2 day trade ban.

**Comment by Zeprix** [comment]
> This is a known issue, and SDA isn't going to get updated for a while. Jessecar96 themselves has said on discord that he's too busy to update SDA right now. I highly recommend that you should use the steam mobile app, if you are able to.
> 
> If you are interested, I maintain steamguard-cli. It's capable of reading the exactly the same maFiles that SDA does, so you don't have to deal with a trade ban. It doesn't have a UI, so if you aren't really comfortable using a terminal, I would recommend you use the steam mobile app. You can download it here: https://github.com/dyc3/steamguard-cli/releases
> 
> Read the quickstart here: https://github.com/dyc3/steamguard-cli/blob/master/docs/quickstart.md
> 
> You can transfer an existing authenticator to the official steam mobile app, but that does cause a 2 day trade ban.

Can it also create maFiles?

**Comment by dyc3** [comment]
Yes, but they won't be compatible with SDA.

**Comment by Zeprix** [comment]
> Yes, but they won't be compatible with SDA.

Will they be the same as generated with mobile authenticator?

**Comment by dyc3** [comment]
No. It's impossible to load the maFiles created by SDA or steamguard-cli into the mobile app, nor vice versa.
